mining/mining-guide.md

```diff
@@ -83,20 +83,21 @@ In order to mine in 2.6, your mining key needs to be present on your machine. If
 
 To maximize your mining efficiency, store data on 4 TB hard drives capable of reading about 200 MiB/s. It is fine to use faster disks, but the extra cost won't be justified.
 
-The first step is to create a folder inside `[data_dir]/storage_modules/` for each of the 4 TB disks you intend to mine with. The Arweave dataset is logically partitioned into a collection of 3.6 TB "mining partitions". Each 4 TB disk will store and mine one of these mining partitions. The partitions are indexed sequentially starting from 0, with 0 being the very first 3.6 TB worth of data stored on Arweave, and ranging to whatever the current mining partition count is. You can choose which mining partitions you store by indicating their index in the folder name. For example, to set up a storage module with the very first mining partition in the weave (packed with your mining address), create the folder \[data_dir]/storage_modules/storage_module_0\_\[your_mining_address].
-
-Mount your drives in the `[data_dir]/storage_modules` folder. E.g.,
+The first step is to create a folder inside `[data_dir]/storage_modules/` for each of the partisions that you intend to mine with. Partition folder names should use the following pattern: `storage_module[_partition_size]_[partition_index]_[your_mining_address]`. As you will see further, specifying `partition_size` is optional. The default partition size is 3.6 TB, as Arweave dataset is logically partitioned into collections of 3.6 TB "mining partitions". For any given `partition_size`, you should take into account around 10% metadata overhead (such as merkle proofs). Hence, for the default 3.6 TB `partition_size`, we recommend a 4 TB disk. The partitions are indexed sequentially starting from 0, with 0 being the very first 3.6 TB (or `partition_size`) worth of data stored on Arweave, and ranging to whatever the current mining partition count is. You can choose which mining partitions you store by indicating their `partition_index` in the folder name. 
```

Comment:
```suggestion
The Arweave dataset is logically partitioned into collections of 3.6 TB "mining partitions". You will store some or all of those mining partitions on your miner in "storage modules". Storage modules can be any size (smaller or larger than the default 3.6TB mining partition size), but many miners opt to align their storage modules with the Arweave partitions.

To setup your storage modules, the first step is to create a folder inside `[data_dir]/storage_modules/` for each of the storage modules that you intend to mine with. Storage module folder names should use the following pattern: `storage_module[_storage_module_size]_[storage_module_index]_[your_mining_address]`. The default storage module size is 3.6TB - in that case specifying `storage_module_size` is optional.

For any given `storage_module_size`, you should allow for an additional 10% metadata overhead (such as merkle proofs). This is why, for the default 3.6 TB `storage_module_size`, we recommend a 4 TB disk. The storage modules are indexed sequentially starting from 0, with 0 being the very first 3.6 TB (or `storage_module_size`) worth of data stored on Arweave, and ranging up to or beyond the current Arweave dataset size (`weave_size`). You can choose which mining partitions you store by indicating their `storage_module_index` in the folder name. 
```

Reply:
Sorry for all the edits. The distinction between storage module and partition is awkward but important... which I think is why I've put off updating the docs.